AndroidUI/Utils/DriverExtensions.cs

```diff
@@ -42,6 +43,34 @@ public static void PressKey(int AndroidKeyCode)
             driver.PressKeyCode(AndroidKeyCode);
         }
 
+        public static void ScrollToElement(By element, ScrollDirection direction)
+        {
+            bool elementFound = false;
+            while (!elementFound)
```

Comment:
I think you need to add limit for amount of scrolls to exit from the cycle if element does not exist.
Also you can avoid using 'result' and 'elementFound' variables by using while(!driver.FindElement(element).Displayed)

Reply:
Sorry, the second part not fully correct, you need to use one bool variable to not break cycle on try-catch. like bool isDisplayed = driver.FindElement(element).Dislpayed